py_gtfs_rt_ingestion/poetry.lock

```diff
@@ -0,0 +1,208 @@
+[[package]]
```

Comment:
this file ensures that all developers are using the same versions of python and package dependencies.